src/items/items.controller.ts

```diff
@@ -2,71 +2,67 @@ import {
   Controller,
   Get,
   Post,
+  Patch,
   Param,
+  Query,
   Body,
-  Delete,
   HttpCode,
 } from '@nestjs/common';
 import { ItemsService } from './items.service';
-import {
-  BuyItemDto,
-  EquipItemDto,
-  UnequipItemDto,
-} from './dto/item-changeStatus.dto';
-import { ApiTags } from '@nestjs/swagger';
+import { CreateItemDto } from './dto/create-item.dto';
+import { UpdateItemDto } from './dto/update-item.dto';
 import { ApiItems } from './items.swagger';
+import { PaginationQueryDto } from './dto/pagination-query.dto';
+import { PositiveIntPipe } from 'src/common/pipes/positive-int/positive-int.pipe';
+import { CategoryQueryDto } from './dto/category-query.dto';
 
-@ApiTags('items')
-@Controller('items') // '/items'경로 요청 처리
+@Controller('items')
 export class ItemsController {
   constructor(private readonly itemsService: ItemsService) {}
 
-  // GET /items 요청 처리
-  @Get()
-  @ApiItems.getAllItems()
-  async getAllItems() {
-    return await this.itemsService.getAllItems(); //service에서 아이템 목록을 받아서, 반환(return)
+  //1. 새로운 아이템 추가 : POST /items
+  @Post()
+  @ApiItems.createItem()
+  @HttpCode(201)
+  async createItem(@Body() createItemDto: CreateItemDto) {
+    return await this.itemsService.createItem(createItemDto);
   }
```

Comment:
```suggestion
  //1. 새로운 아이템 추가 : POST /items
  @Post()
  @ApiItems.createItem()
  @HttpCode(201)
  @UseGuards(AuthGuard('adminAccessToken'))
  async createItem(@Body() createItemDto: CreateItemDto) {
    return await this.itemsService.createItem(createItemDto);
  }
```
item 추가와 삭제 API는 어드민만 사용해야 하기 때문에 어드민 가드를 사용해서 권한을 확인하는 작업이 필요해 보입니다.

Reply:
맞아요 ㅎㅎ 적용했습니다, 감사합니다!